remote/web/package.json

```diff
@@ -8,15 +8,15 @@
     "@vscode/iconv-lite-umd": "0.7.1",
     "@vscode/tree-sitter-wasm": "^0.3.0",
     "@vscode/vscode-languagedetection": "1.0.21",
-    "@xterm/addon-clipboard": "^0.3.0-beta.22",
-    "@xterm/addon-image": "^0.10.0-beta.22",
-    "@xterm/addon-ligatures": "^0.11.0-beta.22",
-    "@xterm/addon-progress": "^0.3.0-beta.22",
-    "@xterm/addon-search": "^0.17.0-beta.22",
-    "@xterm/addon-serialize": "^0.15.0-beta.22",
-    "@xterm/addon-unicode11": "^0.10.0-beta.22",
-    "@xterm/addon-webgl": "^0.20.0-beta.21",
-    "@xterm/xterm": "^6.1.0-beta.22",
+    "@xterm/addon-clipboard": "^0.3.0-beta.45",
+    "@xterm/addon-image": "^0.10.0-beta.45",
+    "@xterm/addon-ligatures": "^0.11.0-beta.45",
+    "@xterm/addon-progress": "^0.3.0-beta.45",
+    "@xterm/addon-search": "^0.17.0-beta.45",
+    "@xterm/addon-serialize": "^0.15.0-beta.45",
+    "@xterm/addon-unicode11": "^0.10.0-beta.45",
+    "@xterm/addon-webgl": "^0.20.0-beta.44",
+    "@xterm/xterm": "^6.1.0-beta.45",
```

Comment:
Yeah let's take this out for now, I'll have a PR coming soon which will update xterm and close off a few VS Code issues so want them all linked there.